doc/configuration/useridresolvers.rst

```diff
@@ -193,14 +193,18 @@ has not passed yet. If the *Per-process server pool* is enabled, knowledge about
 servers is persisted within each process. This setting may improve performance in situations in
 which a LDAP server from the pool is down for extended periods of time.
 
+TLS Version
+~~~~~~~~~~~
+
+When using TLS, you may specify the TLS version to use. Starting from version 3.6, privacyIDEA
+uses TLS v1.3 by default.
```

Comment:
I think we should call it like

"...privacyIDEA offers TLS v1.3 as default value for new resovlers"

otherwise it could sound as if the configuration would have been changed.